packages/server/lib/cypress.coffee

```diff
@@ -110,11 +110,18 @@ module.exports = {
   start: (argv = []) ->
     debug("starting cypress with argv %o", argv)
 
+    options = require("./util/args").toObject(argv)
+
+    if options.runProject and not options.headed
+      # scale the electron browser window
+      # to force retina screens to not
+      # upsample their images when offscreen
+      # rendering
+      require("./util/electron_app").scale()
```

Comment:
@brian-mann Hey, I had an issue in which I couldn't use the image regression plugin `cypress-image-snapshot` because [the snapshots created were different between headed mode and headless mode](https://github.com/cypress-io/cypress/issues/2102#issuecomment-521299946).

I worked out that this was because 9`BrowserWindow#capturePage`](https://github.com/cypress-io/cypress/blob/f89a8236a85dab0cf40b634a4bc5efa101029055/packages/server/lib/gui/windows.coffee#L127-L130) resolves to an image of a different size dependent on the scale factor, and by default `cypress open` was giving me a scale factor which was different from the one set in headless mode.

How would you feel about there being a configuration option to set a scale factor across all browsers? Currently, this hardcodes it as 1, and there isn't a way of setting it for a headed electron environment.